src/test/scala/tech/cryptonomic/conseil/tezos/michelson/parser/JsonParserSpec.scala

```diff
@@ -90,8 +90,8 @@ class JsonParserSpec extends FlatSpec with Matchers {
 
     parse[MichelsonInstruction](json) should equal(Right(
       MichelsonInstructionSequence(List(
-        MichelsonComplexInstruction("DIP", MichelsonInstructionSequence(List(
-          MichelsonSimpleInstruction("DUP"))))))))
+        MichelsonComplexInstruction("DIP", List(MichelsonInstructionSequence(List(
```

Comment:
Trivial comment: using indentation and formatting might make it easier to read such nesting of Lists.

I don't know if using the `a :: b :: Nil` format might make it all more parseable to the reader

Reply:
The alternatives would look something like
```scala
MichelsonInstructionSequence(
  List(
    MichelsonComplexInstruction("DIP", 
      List(
        MichelsonInstructionSequence(
          List(
            MichelsonSimpleInstruction("DUP")
          )
        )
      )
    )
  )
)
//vs.
MichelsonInstructionSequence(
  MichelsonComplexInstruction("DIP",
    MichelsonInstructionSequence(
      MichelsonSimpleInstruction("DUP") :: Nil
    ) :: Nil
  ) :: Nil
)
```

Reply:
I'm refactoring this structure now to be able to add support for LAMBDA instruction and I'll need to change tests anyway so I'll try with ``a :: b :: Nil`` syntax. It might look better indeed.